src/tpt_reports/tpt_reports.py

```diff
@@ -1,15 +1,15 @@
 """cisagov/tpt-reports: A tool for creating Technical Phishing Test (TPT) reports.
 
 Usage:
-  tpt-reports SERVICENOW_ID ELECTION_NAME DOMAIN_TESTED JSON_FILE_PATH OUTPUT_DIRECTORY [--log-level=LEVEL]
+  tpt-reports ASSESSMENT_ID ELECTION_NAME DOMAIN_TESTED JSON_FILE_PATH OUTPUT_DIRECTORY [--log-level=LEVEL]
 
 Options:
   -h --help                         Show this message.
   -l --log-level=LEVEL              If specified, then the log level will be set to
                                     the specified value.  Valid values are "debug", "info",
                                     "warning", "error", and "critical". [default: info]
 Arguments:
-  SERVICENOW_ID                     The ID number in Service Now
+  ASSESSMENT_ID                     The assigned Assessment ID.
```

Comment:
May I suggest a slight rewording here:
```suggestion
  ASSESSMENT_ID                     The assessment identifier.
```